File: app/src/main/java/com/work/ddnet/I.java
package com.work.ddnet;

/**
 * @author Ting
 * @date 2017/11/10
 */

public interface I {
    interface Configkey {
        String CONFIG_READY = "CONFIG_READY";
        String APPLICATION_CONTEXT = "APPLICATION_CONTEXT";
        String API_HOST = "API_HOST";
        String LOADER_DELAYED = "LOADER_DELAYED";
        String HANDLER = "HANDLER";
        String INTERCEPTOR = "INTERCEPTOR";
    }

    interface HttpMethod {
        String GET = "GET";
        String POST = "POST";
        String POST_RAW = "POST_RAW";
        String PUT = "PUT";
        String PUT_RAW = "PUT_RAW";
        String DELETE = "DELETE";
        String UPLOAD = "UPLOAD";
    }

    interface BaseUrl {
        String URL = "http://news.baidu.com/";
    }
}

File: app/src/main/java/com/work/ddnet/I.kt
package com.work.ddnet

/**
 * @author Ting
 * @date 2017/11/10
 */

interface I {
    interface Configkey {
        companion object {
            val CONFIG_READY = "CONFIG_READY"
            val APPLICATION_CONTEXT = "APPLICATION_CONTEXT"
            val API_HOST = "API_HOST"
            val LOADER_DELAYED = "LOADER_DELAYED"
            val HANDLER = "HANDLER"
            val INTERCEPTOR = "INTERCEPTOR"
        }
    }

    interface HttpMethod {
        companion object {
            val GET = "GET"
            val POST = "POST"
            val POST_RAW = "POST_RAW"
            val PUT = "PUT"
            val PUT_RAW = "PUT_RAW"
            val DELETE = "DELETE"
            val UPLOAD = "UPLOAD"
        }
    }

    interface BaseUrl {
        companion object {
            val URL = "http://news.baidu.com/"
        }
    }
}
